hswindow/hseditor/hssoundeditor/soundeditor.go

```diff
@@ -53,7 +53,8 @@ func Create(_ *hsconfig.Config,
 	data *[]byte, x, y float32, project *hsproject.Project) (hscommon.EditorWindow, error) {
 	streamer, format, err := wav.Decode(bytes.NewReader(*data))
 	if err != nil {
-		log.Fatal(err)
+		log.Print(err)
```

Comment:
this `log.Print` should be reported by the caller of `Create`. You don't need to put it here.

Reply:
I didn't put it there, I altered what was there. I was specifically removing all instances of `log.Fatal`.

Reply:
hmm, right, `log` call was here, but you added `return` statement in a line below, so this log call is not necessary anyway ;-)

Reply:
That's a good point. I did both of those edits on two different commits, for two different reasons. One commit was to bubble up errors, another commit was to remove instances of `log.Fatal`.